app/services/record_declarations/retained/ecf.rb

```diff
@@ -3,12 +3,14 @@
 module RecordDeclarations
   module Retained
     class ECF < ::RecordDeclarations::Base
-      include RecordDeclarations::ECF
-
       class << self
         def valid_evidence_types
           %w[training-event-attended self-study-material-completed other].freeze
         end
+
+        def declaration_model
+          ParticipantDeclaration::ECF
+        end
```

Comment:
Could we pass this as this an argument to the service class?

Reply:
sure but not for this change. I don't want to modify the API, i'm merely inlining code from elsewhere so the change is minimal